### DISCUSSION TITLE
Modeling cryogenic vapour and particle expsosure  to solid surface

### DISCUSSION THREAD

**Discussion by Muthu-V** [firemodels/fds#13792] [Q&A]
Dear all,
                Since FDS is originally made for Fires, I see some clipping values are done for density and temperatures . If I wanted to run vapor jets and sublimating solid particle jets at cryogenic conditions, I presume I need to explicitly specify the Temperature ranges for the solid surfaces not to clip at default settings. Are there anything more to be checked and specified when we extend the code usage for such applications Ex: sublimating CO2 solid particles , LNG  etc.
Thanks

**Comment by drjfloyd** [comment]
FDS will adjust the upper or lower clipping temperature to include any INIT temperature, TMP_FRONT for any SURF or the melting temperature for any liquid SPEC. Clipping for high density is set to twice the density of the highest MW tracked species at the clipping temperature. It is density clipping that is actually used in the transport routines.

**Reply by Muthu-V** [reply]
Thank you for your Clarification.
